prow/spyglass/lenses/podinfo/template.html

```diff
@@ -24,6 +25,10 @@
         <td class="mdl-data-table__cell--non-numeric">Pod name</td>
         <td class="mdl-data-table__cell--non-numeric literal">{{$pod.Name}}</td>
       </tr>
+      <tr>
+        <td class="mdl-data-table__cell--non-numeric">Pod Link</td>
+        <td class="mdl-data-table__cell--non-numeric literal">{{$podLink}}</td>
+      </tr>
```

Comment:
We should omit this row if the pod link is not specified, especially since that will be the default and common case.

Reply:
yeah that's right, done